server/src/main/java/org/elasticsearch/index/mapper/MapperException.java

```diff
@@ -33,11 +33,15 @@ public MapperException(String message) {
         super(message);
     }
 
+    public MapperException(String msg, Object... args) {
+        super(msg, args);
+    }
```

Comment:
nit: not sure we need another constructor here, can you just insert the args into the message String in the call?

Reply:
Sure.
Is `LoggerMessageFormat#format` preferred over `String#format` or maybe just string concatenation?